src/funcapp.jl

```diff
@@ -1,74 +1,304 @@
-	
+using FastGaussQuadrature
+using Gadfly
+using Base.Test
+using ApproxFun
+using ApproXD
+using Plots
 
```

Comment:
Apologies for the mistake. I've made a commit with the change and also commented savefig() for the first 2 functions since this resulted in an error message when I ran `include("run.jl")`.